### ISSUE TITLE
Add ability to set anchors/offsets for lines and pathtexts

### ISSUE THREAD

**Issue by pastk** [organicmaps/organicmaps#7952]
It'll be helpful for many cases, e.g.
- render segregated shared paths for cyclists (a cycling line) and pedestrians (white dashed line) https://github.com/organicmaps/organicmaps/pull/7878#issuecomment-2056368770 or trails + ski pistes
- render area borders, e.g. for nature reserves, with visible inner and outer sides https://github.com/organicmaps/organicmaps/issues/7812#issuecomment-2038006166
- one-sided barriers, etc.

Similar for pathtexts, we'd want to render area's name on the inner side of the border.
- also for #7464 

Basically its enough to have an `anchor: left/right` attrib. If its present the renderer should offset the line so that it runs exactly to the left/right of the "main" line (if present) or the default centerline.
If we add an ability to set both an anchor and an offset value, then it'd be possible to render slightly separated lines or partially overlapping ones, e.g.
![image](https://github.com/organicmaps/organicmaps/assets/18434508/d6fa042e-475e-40d8-822e-5ca764e2583f)
Fancy steps? :) But maybe we don't need it.

FYI @vng

**Comment by pastk** [comment]
It might be tricky for thicker lines though (e.g. for "fading" borders)
The turns might not look good unless specially processed
![image](https://github.com/organicmaps/organicmaps/assets/18434508/63eee500-596e-4331-8eee-8afc7d737b3e)

**Comment by euf** [comment]
It would be also nice to render sidewalk:side=* mapped on the main road. 

https://wiki.openstreetmap.org/wiki/Key:sidewalk

The same applies for similar tagging of cycleways, bus lanes, etc. Like this: 

![image](https://github.com/organicmaps/organicmaps/assets/11754264/9a0609f6-1f10-4078-af67-e72c12cfddf7)

Source: https://imagico.de/blog/en/the-world-of-parking-3-some-odds-and-ends/

**Comment by biodranik** [comment]
A user either needs to see cycleways (rides a bicycle) or doesn't need them (doesn't ride). By using a separate style any cycling-compatible routes can be rendered prominently. And be completely turned off by the general style.
